custom-prom.md

```diff
@@ -146,22 +146,67 @@ Make sure that [honor_labels](https://prometheus.io/docs/prometheus/latest/confi
 
 ### Negative idle reported
 
+#### Single Cluster Tests
+
+Ensure results are not null for both queries below.
+
+Bad:
+
+```json
+{"status":"success","data":{"resultType":"vector","result":[]}}
+```
+
+Good:
+
+```json
+{"status":"success","data":{"resultType":"vector","result":[{"metric":{"__name__":"node_total_hourly_cost","instance":"aks-agentpool-81479558-vmss000001","instance_type":"Standard_B4ms","job":"kubecost","node":"aks-agentpool-81479558-vmss000001","provider_id":"azure:///subscriptions/0bd50fdf-c923-4e1e-850c-196dd3dcc5d3/resourceGroups/mc_kc-demo_kc-demo-dev_eastus/providers/Microsoft.Compute/virtualMachineScaleSets/aks-agentpool-81479558-vmss/virtualMachines/1","region":"eastus"},"value":[1673020150,"0.16599565032196045"]}]}}
```

Comment:
Should we hide the Azure subscription ID ? `0bd50fdf-c923-4e1e-850c-196dd3dcc5d3`